src/ipc_internal.h

```diff
@@ -21,7 +21,7 @@ extern "C" {
 // providerIpcData is a Flexible Array Member because its size varies
 // depending on the provider.
 typedef struct umf_ipc_data_t {
-    size_t size; // size of base allocation
+    size_t base_size; // size of base (coarse-grain) allocation
```

Comment:
nit: the naming scheme is inconsistent with other member variables

Reply:
Fixed.